.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Recipes added with this pull request (max 5):
+ 
+ - package1
+ - package2
+ 
+I have checked the following in my recipes:
+* [ ] My files end with only 1 newline character, no more, no less.
+* [ ] My recipe follow the same style as [example](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml) as much as possible.
+* [ ] The license field also specifies the number of the license if applicable (e.g. `GPLv3` instead of `GPL` or `BSD 3-clause` instead of `BSD`).
+* [ ] My recipe has tests.
```

Comment:
Should we be a bit more specific about what acceptable tests are? For instance, running `make check`, `ctest`, what have you. Also, verifying all files were installed properly. Possibly, a simple program to exercise the package (especially if there is no `make check`, etc.).